doc/release-notes.md

```diff
@@ -1,9 +1,9 @@
-27.1 Release Notes
+27.x Release Notes
 =====================
 
-Bitcoin Core version 27.1 is now available from:
+Bitcoin Core version 27.x is now available from:
```

Comment:
I know we usually do it this way, but what's the harm in just making this (and other references, except for the git tag) `27.2rc1` right away?

Reply:
The next release might not be 27.2. It could be 27.1.1 etc.

Reply:
Sure, but that seems to be an undocumented exception? Since 0.10.0, it [seems](https://bitcoincore.org/bin/) we only have 3 (in the root dir) releases where we we had a non-zero patch number. We also don't mention patch versions in our [documentation](https://bitcoincore.org/en/lifecycle/).

Keeping the git tag to `.x` makes sense because it gives us the flexibility to release as a minor or a patch version, i.e. we can still easily update the release notes to `27.1.1` if necessary. It just seems like currently the code/doc updates just add a bit of unnecessary churn? Anyway, no biggie either way.

Reply:
Ok. Lets follow up with ways we might want to streamline this for `28.x`.